docs/integrations/blueprism.mdx

```diff
@@ -0,0 +1,130 @@
+
+All you need to get started is:
+- Blue Prism is installed on your computer.
+- Account on Cradl AI (If you don't have one yet, you can sign up [here](https://app.cradl.ai/login?redirect=signup&utm_campaign=docs).
+
+
+:::tip Scalable
+Cradl AI empowers you with the ability to create your own AI models. 15 examples are all you need to get started.
+:::
+
+<br/>
+
+## Overview
+
+<br/>
+The Cradl AI asset consists of these components:
+
+| Name          | Type | Description |
+|:------------| :------------- |:--- |
+| Cradl AI           | Visual Business Object (VBO) | This is where all the actions for this guide can be found. |
+| Cradl Demo Process | Process | A demo process that shows an example of how you can retrieve the parsed documents from your Flow in Cradl AI. |
+| Cradl AI Web API   | Web Api Service | The API itself, for advanced cases that require customization it is possible to make adjustments here to get the most out of Cradl AI. |
+| Cradl File Server  | Web Api Service | To support all sorts of documents of different sizes, Cradl AI comes with a file server that is used behind the scenes, it is not recommended to make any changes in this component. |
+| Cradl Credentials  | OAuth 2.0 - Client Credentials | Make sure to add your own Client ID and Client Secret in this object for everything to play along nicely. |
+
+<br/>
+
+## Step 1: Login to Cradl and setup a Flow
+
+<br/>
+
+There is very little setup required in Cradl before you are able to adding actions in your Blue Prism processes.
+All you have to do is:
+
+- Create a new flow for your documents [here](https://app.cradl.ai/flows).
+- Select an initial model just to get started, you are free to customize the model to extract the fields you want. 
+- Add your co-workers as human validators to get help with the initial validation process.
+
+You are now ready to jump into Blue Prism and start the integration.
+But on your way there, remember to bring the parameters that are listed under Blue Prism in input or export dropdown.
+They are essential to identify both you, your model and your flow. Here is a brief explanation of what they are and how they are used:
+
+| Parameter     | Type | Description |
+|:--------------| :------------- |:--- |
+| Client ID     | Credentials | The identity of your Blue Prism integration. Input this along with Client Secret when prompted with credentials. |
+| Client Secret | Credentials | The secret key to authenticate your client. do not share this with anyone, but store it safely when prompted with credentials during import. |
+| flowId        | Data Object | The identity of the very flow you just created. It is used as an input to some of the actions in the VBO. |
+| modelId       | Data Object | The identity of the model you are using in this flow. It is used as an input to some of the actions in the VBO. |
+
+
+:::tip Adaptive
+You can at any point change the model in your flow by selecting another modelId.
```

Comment:
I am afraid this may lead to more confusion. When the modelId in the flow is changed in cradl, that change is automatically reflected in BluePrism as well. Unless they are using the predictions endpoint explicitly, they don't have to change the modelId.